File: jats-demo-webapp/src/main/ts/model/Product.ts
class Product
{
	private variants : Component[];

	private id : number;

	private label : string;

	public getVariants() { return this.variants; }
	public setVariants( variants : Component[] ) { this.variants = variants; }

	public getId() { return this.id; }
	public setId( id : number ) { this.id = id; }

	public getLabel() { return this.label; }
	public setLabel( label : string ) { this.label = label; }


}

File: jats-demo-model/src/main/java/com/github/pepe79/jats/demo/model/Product.java
package com.github.pepe79.jats.demo.model;

import java.util.List;

public class Product
{
	private int id;
	
	private String label; 

	private List<Component> variants;

	public List<Component> getVariants()
	{
		return variants;
	}

	public void setVariants(List<Component> variants)
	{
		this.variants = variants;
	}

	public int getId()
	{
		return id;
	}

	public void setId(int id)
	{
		this.id = id;
	}

	public String getLabel()
	{
		return label;
	}

	public void setLabel(String label)
	{
		this.label = label;
	}

	
}
